net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/mods-enabled-sql

```diff
@@ -0,0 +1,46 @@
+{% if helpers.exists('OPNsense.freeradius.general.enabled') and OPNsense.freeradius.general.enabled == '1' %}
+{%   if helpers.exists('OPNsense.freeradius.general.sqlite') and OPNsense.freeradius.general.sqlite == '1' %}
+
+sql {
+        driver = "rlm_sql_sqlite"
+        sqlite {
+                # Path to the sqlite database
+                filename = "/tmp/freeradius.db"
```

Comment:
@mimugmail I think the database itself should be outside /tmp/, the session data won't survive a reboot.

Reply:
Something like clamav uses in the rc.conf:

clamav_clamd_var_mfs="/var/db/clamav"